src/libraries/System.Drawing.Common/src/System/Drawing/Gdiplus.cs

```diff
@@ -178,6 +178,7 @@ public const int
         DC_PAPERNAMES = 16,
         DC_ORIENTATION = 17,
         DC_COPIES = 18,
+        DC_COLORDEVICE = 32,
```

Comment:
nit: adding a comment would be good

```
If the printer supports color printing, the return value is 1; otherwise, the return value is zero. The pOutput parameter is not used.
```

Reply:
or have the comment on the calling site.